### ISSUE TITLE
Usage of lodash "get" and "set" functions leads to memory leaks

### ISSUE THREAD

**Issue by MaciejBaj** [LiskHQ/wamp-socket-cluster#24]
Both `lodash.set` and `lodash.get` functions are caching results for the accessed paths - 
```
const accessedValue = get(exampleObject, 'a.b.c.d.X');
```
When `X` variable in path will be different every time, `lodash` will keep caching every `accessedValue` which was accessed previously, call stack:

1. `get` calls `baseGet` https://github.com/lodash/lodash/blob/4.4.2-npm-packages/lodash.get/index.js#L927
2. `baseGet` calls `castPath` https://github.com/lodash/lodash/blob/4.4.2-npm-packages/lodash.get/index.js#L458
3. `castPath` calls `stringToPath` https://github.com/lodash/lodash/blob/4.4.2-npm-packages/lodash.get/index.js#L513
4. `stringToPath` calls `memoize` https://github.com/lodash/lodash/blob/4.4.2-npm-packages/lodash.get/index.js#L597
5. `memoize` caches every key which wasn't found https://github.com/lodash/lodash/blob/4.4.2-npm-packages/lodash.get/index.js#L701

As the paths are unique by design for every RPC call, all of them are being stored inside of the lodash library as the part of `memoized.cache` object.

The issue was already reported - https://github.com/lodash/lodash/issues/3307 
`lodash.get` and `lodash.set` functions installed as separate npm packages are keep using `memoize` instead of safe `memoizedCapped`, which stores values in the cache if the limit of 500 entries in not reached. 

For purposes of `wamp-socket-cluster`, the functionality of caching results is totally irrelevant and the lodash `get` and `set` functions should be replaced by its equivalents without caching the results.

**Comment by alepop** [comment]
@MaciejBaj I hope this helps you
```
const getKeys = path => Array.isArray(path) ? path : path.split('.');

const get = (obj, path, defaultVal) => {
   const [head, ...tail] = getKeys(path);
    obj = obj[head];
    if (obj && !!tail.length) {
        return get(obj, tail, defaultVal);
    }
    return obj ?obj : defaultVal;
}

const set = (obj, path, val ) => {
 const [head, ...tail]  = getKeys(path);
  if (!!tail.length) {
    obj[head] = obj[head] || {};
    return set(obj[head], tail, val );
  }
 obj[head] = val;
}
```

**Comment by MaciejBaj** [comment]
Thanks, @alepop, I've noticed your comment too late. I've opened the PR (https://github.com/LiskHQ/wamp-socket-cluster/pull/25), I will appreciate your review.

**Comment by MaciejBaj** [comment]
Closed by #25
